clusters/app.ci/supplemental-ci-images/libguestfs-tools.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: image.openshift.io/v1
```

Comment:
No. Please follow the process documented in https://docs.ci.openshift.org/docs/how-tos/external-images/ to see how to import external images to CI cluster.

Reply:
Seems like you already did that in https://github.com/openshift/release/pull/25423#issuecomment-1016906942 so this part of this PR can simply be removed and then you do not need TP to approve.

/uncc
/unassign @smg247

Reply:
Sorry, this is leftover from another PR. I will remove it.